lib/matplotlib/backends/_backend_tk.py

```diff
@@ -718,7 +722,7 @@ def set_cursor(self, cursor):
 
 
 class ToolbarTk(ToolContainerBase, tk.Frame):
-    _icon_extension = '.gif'
+    _icon_extension = '.png'
```

Comment:
Do we still need that variable or can we inline since we only use png anyway? (Can't check easily, because I'm on mobile.)

Reply:
This is used by `backend_bases` and the gtk3 code path has this value as `_icon_extension = '-symbolic.svg'`, however we can remove this line as the baseclass sets this value to png so we can remove this line.

```suggestion
```

Reply:
Good point, removed it.